ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -113,73 +113,95 @@ host_monitoring_cron:
 - name: send pcp ping every 5 mintues
   minute: "*/5"
   job: ops-runner -f -s 60 -n cspp.pcp.ping /usr/bin/cron-send-pcp-ping
+# REMOVE_IH: I'm not sure I see the point of this if we are running pcp checks every minute,  
```

Comment:
Decision: Remove this check script, the pcp.ping item from the template and cron job.  Change the NODATA trigger to use kernel.uname.sysname item.